assets/app/view/game/round/stock.rb

```diff
@@ -375,6 +376,46 @@ def render_bid_input(company)
 
           [h(Bid, entity: @current_entity, corporation: company)]
         end
+
+        def render_buyable_minors
+          props = {
+            style: {
+              display: 'inline-block',
+              verticalAlign: 'top',
+            },
+          }
+
+          @step.buyable_minors.map do |minor|
```

Comment:
Very, very minor nit: purchasable_companies vs. buyable_minors

Reply:
Have now renamed to purchasable_minors, and also use respond_to? for it, to avoid affecting other titles.